src/Resource/Listing.php

```diff
@@ -300,4 +300,266 @@ public function deletePaymentMethod($listingId, $paymentMethodId)
 
         return $response->getStatusCode() === 204;
     }
+
+    /**
+     * @param int $listingId
+     * @return \Traum\Entity\ListingLeisureActivityCollection
+     */
+    public function getLeisureActivities($listingId)
+    {
+        $uri = sprintf('/listing/%d/leisure-activity', $listingId);
+        $body = $this->executeGet($uri);
+
+        return new Entity\ListingLeisureActivityCollection($body);
+    }
+
+    /**
+     * @param int                                  $listingId
+     * @param \Traum\Entity\ListingLeisureActivity $listingLeisureActivity
+     * @return \Traum\Entity\ListingLeisureActivity
+     * ToDo: post collection or entity? (https://github.com/theDisco/traum/issues/7)
+     */
+    public function postLeisureActivities($listingId, Entity\ListingLeisureActivity $listingLeisureActivity)
```

Comment:
This should be singular (`public function postLeisureActivity`), since you can only post one entity.